frontend/packages/topology/src/elements/BaseGraph.ts

```diff
@@ -25,6 +25,8 @@ export default class BaseGraph<E extends GraphModel = GraphModel, D = any> exten
 
   private currentLayout?: Layout;
 
+  private zoomExtent: [number, number] = [0.25, 4];
```

Comment:
Is there a type here? What are the two numbers representing? Min & Max? Maybe a type of `ZoomBounds`?